internal/postgres/utils.go

```diff
@@ -20,7 +20,71 @@ type PgxArray[T any] struct {
 // properly handles scanning postgres arrays
 func (a *PgxArray[T]) Scan(src any) error {
 	m := pgtype.NewMap()
-	pgt, ok := m.TypeForName(strings.ToLower(a.colDataType))
+	// Register money types
+	m.RegisterType(&pgtype.Type{
+		Name:  "money",
+		OID:   790,
+		Codec: pgtype.TextCodec{},
+	})
+	m.RegisterType(&pgtype.Type{
+		Name: "_money",
+		OID:  791,
+		Codec: &pgtype.ArrayCodec{
+			ElementType: &pgtype.Type{
+				Name:  "money",
+				OID:   790,
+				Codec: pgtype.TextCodec{},
+			},
+		},
+	})
+
+	// Register UUID types
+	m.RegisterType(&pgtype.Type{
+		Name:  "uuid",
+		OID:   2950, // UUID type OID
+		Codec: pgtype.TextCodec{},
+	})
+
+	m.RegisterType(&pgtype.Type{
+		Name: "_uuid",
+		OID:  2951,
+		Codec: &pgtype.ArrayCodec{
+			ElementType: &pgtype.Type{
+				Name:  "uuid",
+				OID:   2950,
+				Codec: pgtype.TextCodec{},
+			},
+		},
+	})
+
+	// Register XML type
+	m.RegisterType(&pgtype.Type{
```

Comment:
Same Q here, they have a `pgtype.XMLCodec`, what is the diff?

Reply:
Definitely doesnt look like there is an explicit xml scanner but still confused on the UUID one

Reply:
same thing I want the xml as a string instead of the xml struct